New package: AnyLogic.AnyLogic.University version 8.7.12

mdanish-kh opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66122)

Commit:
New package: AnyLogic.AnyLogic.Uni version 8.7.12

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+- Architecture: x64
+  InstallerUrl: https://files.anylogic.com/anylogic-university-8.7.12.x86_64.exe
+  InstallerSha256: 3EE743DACD203BE040FB4014672257864310B94C49996720684794A773752EF0
+- Architecture: x86
+  InstallerUrl: https://files.anylogic.com/anylogic-university-8.7.12.x86_64.exe
+  InstallerSha256: 3EE743DACD203BE040FB4014672257864310B94C49996720684794A773752EF0
```
```suggestion
```
![image](https://user-images.githubusercontent.com/12611259/178645656-1d8ef6fc-061b-4a72-aeca-f5d54a8baa85.png)

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.Uni
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.University
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.Uni
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.University
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.yaml:
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.Uni
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.University
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# PublisherSupportUrl: 
+PrivacyUrl: https://anylogic.com/upload/privacy_policy/anylogic-privacy-policy.pdf
+# Author: 
+PackageName: Anylogic University
```
Because they put the version in the middle of the display name, AppsAndFeaturesEntries are needed

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+FileExtensions:
+- alp
+- java
+ReleaseDate: 2022-05-25
```
```suggestion
ReleaseDate: 2022-05-25
AppsAndFeaturesEntries: 
- DisplayName: AnyLogic 8.7.12 University
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# CopyrightUrl: 
+ShortDescription: AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company.
+Description:  AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company. It supports agent-based, discrete event, and system dynamics simulation methodologies.
+Moniker: anylogic
```
```suggestion
Moniker: anylogic-university
```

mdanish-kh replied:
Isn't it more convenient to choose the version when user types `winget install anylogic` (as this is more easy to remember) ? With this change, I assume this university version won't show up when user types the above command. Also, with this change, will this version show up when user types `winget search anylogic` ? Should we add anylogic in its tags?

Commit:
Fix AnyLogic.AnyLogic.University version 8.7.12

- Change package name 'Uni' to 'University'
- Remove x86 installer entry
- Add AppsAndFeaturesEntries
- Rename directory and manifest files

Commit:
Update moniker 'anylogic' to 'anylogic-university'.
